README.md

```diff
@@ -22,20 +22,31 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+## Getting started
+For an overview of Palisade, start with the Palisade introduction and the accompanying guides: QuickStart Guide; and Developer Guide: [Palisade README](https://github.com/gchq/Palisade).
+
 # Palisade Clients
 
 ## Client Implementations
+Implementation of a client will need to send requests into Palisade and process the response.  
+The workflow for this is expected to follow the pattern of first sending in a request to register access for data.  
+This will return a response consisting of a token (a unique id) that can then be used in a second request to retrieve the processed data.
+An implementation of a Client will need to conform to this workflow, but the technology that is used can be left to best suit the intended use of the application.
```

Comment:
```suggestion
The Client implementations will need to send requests into Palisade and process the response.  
The workflow for this is expected to follow the pattern of first sending in a request to register access for data.  
This will return a response consisting of a token (a unique id) that can then be used in a second request to retrieve the actual data.
An implementation of a Client will need to conform to this workflow, but the technology that is used will be best suited to the intended use of the application.
```